Test/FMI3/fmi3_import_variable_test.cpp

```diff
@@ -321,7 +321,7 @@ TEST_CASE("Invalid Binary variable - non-hexadecimal char first in byte tuple",
     const char* xmldir = FMI3_TEST_XML_DIR "/variable_test/invalid/binaryStart1";
 
     fmi3_import_t* xml = fmi3_testutil_parse_xml(xmldir);
-    REQUIRE(xml != nullptr);
+    REQUIRE(xml != nullptr); /* TODO how do we want this to work? */
```

Comment:
Only remaining action is to figure out how we want to handle this scenario, currently the tests fail on this branch, but the tests only fail here.